### ISSUE TITLE
Changing the syntax highlighting when moving a tab

### ISSUE THREAD

**Issue by dmbond** [JetBrains/resharper-unity#2127]
I have a Unity project under UWP with "Preferences > External Tools > Player projects" enabled
Then I open the file in Rider via Explorer > Solution > Assembly-CSharp-firstpass.Player > FileName.cs and see the highlighted code enclosed in #if ENABLE_WINMD_SUPPORT ... #endif but only until I move tab with this code between tab groups. After moving a tab, code highlighting disappears, as if I just opened this file, not through special project .Player
![first](https://user-images.githubusercontent.com/15363149/126506428-6ff2c307-d76c-4f0b-8c9e-097c44e09ab6.png)
![after_move](https://user-images.githubusercontent.com/15363149/126506449-a1637a99-b616-4a23-885b-fd9025460661.png)

**Comment by van800** [comment]
You may switch context via this in the status bar
![image](https://user-images.githubusercontent.com/1482681/126507862-52af3cec-783b-4269-baaa-24784dcd6072.png)

**Comment by dmbond** [comment]
Yes, this solves the problem, but it is wrong that the tab loses context when moved
